src/RazorSdk/Tool/ServerProtocol/ServerConnection.cs

```diff
@@ -77,13 +77,7 @@ public static string GetTempPath(string workingDir)
                 }
             }
 
-            var userProfile = Environment.GetEnvironmentVariable("USERPROFILE");
-            if (Path.IsPathRooted(userProfile))
-            {
-                return userProfile;
-            }
-
-            return Environment.GetEnvironmentVariable("SYSTEMROOT");
+            return Environment.GetFolderPath(Environment.SpecialFolder.UserProfile);
```

Comment:
I don't understand the motivation for this change. The PR description references `GetDotnetHomePath` but here we are resolving a temp path, so that seems unrelated. I see `SpecialFolder.UserProfile` is equivalent to `%USERPROFILE%`, but it's not clear to me why we removed the `%SYSTEMROOT%` fallback?